msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -260,13 +260,34 @@ To share the entire app to stage, in the app manifest, you must configure `meeti
 > [!NOTE]
 > For your app to pass validation, when you create a deep link from your website, web app, or Adaptive Card, use **Share in meeting** as the string or copy.
 
+## Build in-meeting document signing app
+
+You can build an in-meeting app for enabling meeting participants to sign documents in real time. It facilitates reviewing and signing documents in a single session. The participants can sign the documents using their current tenant identity.
+
+You can use an in-meeting signing app to:
+
+- Add documents to be reviewed during a meeting
+- Share documents to be reviewed to main stage
+- Sign documents using the signer’s identity
+
+The participants can review and sign documents, such as purchase agreements and purchase orders.
+
+![in-meeting document signing app](~/assets//images/sbs-inmeeting-doc-signing/signing-clip.gif)
+
+The following participant roles may be involved during the meeting:
+
+- **Document creator**: This role can add their own documents to be reviewed and signed.
+- **Signer**: This role can sign reviewed documents.
+- **Reader**: This role can view the documents added to the meeting.
+
 ## Code sample
 
 |Sample name | Description | C# | Node.js |
 |----------------|-----------------|--------------|----------------|
 |Meeting stage sample | Sample app to show a tab in meeting stage for collaboration | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-stage-view/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-stage-view/nodejs) |
 | In-meeting notification | Demonstrates how to implement in-meeting notification using bot. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-content-bubble/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-content-bubble/nodejs) |
 | In-meeting document signing | Demonstrates how to implement a document signing Teams app. Includes sharing specific app content to stage, Teams SSO and user specific stage view. | [View](https://github.com/officedev/microsoft-teams-samples/tree/main/samples/meetings-share-to-stage-signing/csharp) | NA |
+| In-Meeting Document Signing | Sample app that demonstrates the scenario of enabling users to sign documents in real time during meetings making possible to review and sign documents in a single session. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-share-to-stage-signing/csharp) | NA |
```

Comment:
I already have a link to this sample in this table:

| In-meeting document signing | Demonstrates how to implement a document signing Teams app. Includes sharing specific app content to stage, Teams SSO and user specific stage view. | [View](https://github.com/officedev/microsoft-teams-samples/tree/main/samples/meetings-share-to-stage-signing/csharp) | NA |